src/tests/ftest/pool/info_tests.py

```diff
@@ -21,128 +21,122 @@
-                             "expected number, 1. num targets: "
-                             "{0}".format(pool_info.pi_ntargets))
-            self.fail("found number of targets in pool did not match "
-                      "expected number, 1. num targets: "
-                      "{0}".format(pool_info.pi_ntargets))
-
-        # number of disabled targets
-        if pool_info.pi_ndisabled > 0:
-            self.d_log.error("found disabled targets, none expected to be")
-            self.fail("found disabled targets, none expected to be disabled")
-
-        # mode
-        if pool_info.pi_mode != mode:
-            self.d_log.error("found diff mode than expected. expected {0}, "
-                             "found {1}.".format(mode, pool_info.pi_mode))
-            self.fail("found different mode than expected. expected {0}, "
-                      "found {1}.".format(mode, pool_info.pi_mode))
-
-        # uid
-        if pool_info.pi_uid != uid:
-            self.d_log.error("found actual pool uid {0} doesn't match expected "
-                             "uid {1}".format(pool_info.pi_uid, uid))
-            self.fail("found actual pool uid {0} does not match expected uid "
-                      "{1}".format(pool_info.pi_uid, uid))
-
-        # gid
-        if pool_info.pi_gid != gid:
-            self.d_log.error("found actual pool gid {0} doesn't match expected "
-                             "gid {1}".format(pool_info.pi_gid, gid))
-            self.fail("found actual pool gid {0} does not match expected gid "
-                      "{1}".format(pool_info.pi_gid, gid))
+        # Get the test params
+        self.pool = TestPool(self.context, self.log)
+        self.pool.get_params(self)
+        permissions = self.params.get("permissions", "/run/test/*")
+        targets = self.params.get("targets", "/run/server_config/*")
+        pool_targets = len(self.hostlist_servers) * targets
+
+        # Create a pool
+        self.pool.create()
+
+        # Connect to the pool
+        self.pool.connect(permissions)
+
+        # Verify the pool information
+        checks = {
+            "pi_uuid": self.pool.uuid,
+            "pi_ntargets": len(self.hostlist_servers) * targets,
+            "pi_nnodes": len(self.hostlist_servers),
+            "pi_ndisabled": 0,
+            "pi_map_ver": 1,
+            "pi_leader": 0,
+            "pi_bits": 0xFFFFFFFFFFFFFFFF,
+        }
+        status = self.pool.check_pool_info(**checks)
+        self.assertTrue(status, "Invlaid pool information detected prior")
+        checks = {
+            "s_total": (self.pool.scm_size.value, 0),
+            "s_free": (self.pool.scm_size.value - (256 * pool_targets), 0),
+        }
+        status = self.pool.check_pool_daos_space(**checks)
+        self.assertTrue(status, "Invlaid pool space information detected")
+        self.log.info("Test Passed")
+
+        # mode = self.params.get("mode", "/run/pool/*")
```

Comment:
All these commented lines can be removed if not needed.

Reply:
Thanks - fixed